docs/concepts/MSBuild-props-and-targets.md

```diff
@@ -0,0 +1,74 @@
+---
+title: MSBuild props and targets in a package
+description: Describes MSBuild props and targets in NuGet packages
+author: nkolev92
+ms.author: nikolev
+ms.date: 07/13/2022
+ms.topic: conceptual
+---
+
+# MSBuild props and targets in a package
+
+In additional to the more traditional assemblies, NuGet packages may sometimes add custom build targets or properties to projects that consume that package.
+This can be achieved by adding a valid MSBuild file, in the form `<package_id>.targets` or `<package_id>.props` (such as `Contoso.Utility.UsefulStuff.targets`) within the build folders of the project.
+
+## Build folders
+
+As NuGet has evolved, various different folders for build props and targets have been added.
+
+| Folder | NuGet Version | Use |
+|--------|---------------|-----|
+| build | 2.5+ | Build logic for every framework of a project. |
+| buildMultiTargetting | 4.0+ | Build logic for the `outer build` for projects that target multiple frameworks. PackageReference only. |
+| buildTransitive | 5.0+ | Build logic for assets that flow transitively to any consuming project. See the [feature](https://github.com/NuGet/Home/wiki/Allow-package--authors-to-define-build-assets-transitive-behavior) page. PackageReference only. |
+
+## Framework specific build folder

```

Comment:
I don't see anything on the page saying that `build/<packageid>.props` is also possible. I was looking for it, because I think it's important to warn package authors that this makes their package compatible with all projects, so there might be scenarios (like an otherwise .NET package being installed in a C++ project) where the props file will be imported, but not any `lib/<tfm>/*` assets.  At least, that's my understanding of how package compatibility works.

Basically, I want to discourage package authors from using build files without a TFM, but I guess that not documenting it possibly a strong discouragement too 😁

Reply:
2nd sentence says that. 
I've added some more clarifications.